package/boot/uboot-sunxi/patches/063-fix-lime2-revK-add-micrel-PHY.patch

```diff
@@ -11,7 +11,7 @@
  CONFIG_AHCI=y
```

Comment:
This change in `configs/A20-OLinuXino-Lime2-eMMC_defconfig` is already include in upstream U-Boot, see https://github.com/u-boot/u-boot/commit/22885f4ed9f5a67564aa8f0401669a9601051251

The `configs/A20-OLinuXino-Lime2_defconfig` file contains options to activate a Realtek PHY in upstream U-Boot, see https://github.com/u-boot/u-boot/blob/master/configs/A20-OLinuXino-Lime2_defconfig#L23
It only has one Ethernet port, so it is probably using Realtek instead. 

I think you can drop this patch completely.

Reply:
Dropped, indeed this was upstreamed.